doc/user_guide.rst

```diff
@@ -0,0 +1,34 @@
+**********
+User Guide
+**********
+
+Requirements
+============
+
+- Python 3.8
+
+Installing
+==========
+
+.. code-block:: bash
+
+   pip install conda_vulnerability_scanner
+
+Fetching the CVE database
+=========================
+
+.. code-block:: bash
+
+   python3 -m conda_vulnerability_scanner.scanner fetch --db-directory DB_DIRECTORY
```

Comment:
Is DB_DIRECTORY a placeholder for a directory path?
```suggestion
   python3 -m conda_vulnerability_scanner.scanner fetch --db-directory <DB_DIRECTORY>
```

Reply:
here is it a placeholder and usually, I also use the <> around it, but curiously the help of the python command line parser produces it without and I thought, that lets use the syntax the parser produces.